packages/web-components/fast-element/src/observation/observable.ts

```diff
@@ -377,6 +377,12 @@ export interface BindingObserver<TSource = any, TReturn = any, TParent = any>
      * Unsubscribe from all dependent observables of the binding.
      */
     disconnect(): void;
+
+    /**
+     * Gets {@link SubscriptionRecord|SubscriptionRecords} that the {@link BindingObserver}
+     * is observing.
+     */
+    subscriptionRecords(): IterableIterator<SubscriptionRecord>;
```

Comment:
@EisenbergEffect I wasn't exactly sure what to name this so if you have something better in mind let me know. I also thought about simply `BindingObserver.records()`

Reply:
Let's go with `records` if that's ok with you.

Reply:
Also, I think we may need to export `SubscriptionRecord` or some version of it. Do we want to export that directly or have a version that we export with only the `propertySource` and `propertyName` properties? Should we include `notifier` as well?

Reply:
The the `notifier` can get obtained from `Observable.getNotifier(propertySource)` right? If so I think we can probably just expose `propertySource` and `propertyName`

Reply:
@EisenbergEffect changed method name to `records()` and exported an `ObservationRecord` interface